storage.yaml

```diff
@@ -491,6 +491,12 @@ components:
           nullable: true
           format: uri
           example: http://api.example.org/accounts/?page=2
+        continuation_token: 
```

Comment:
Do we need to inform all of these values??

Reply:
Are you talking about the previous fields?

Reply:
I mean : type: string
          nullable: true
          maxLength: 200
          minLength: 10
          pattern: '.*'

Reply:
lint required 😄

Reply:
:this-is-fine-fire: